### ISSUE TITLE
Having an issue where player count only goes up when players are in the highest priority 

### ISSUE THREAD

**Issue by Spreax** [PaperMC/Velocity#963]
Keep in mind I am using Ambassador on 1.16.5 Arclight servers

So I'm using one server that is my Lobby and another that is my Modded Pixelmon server. The priority is set in this order.

<img width="553" alt="image" src="https://user-images.githubusercontent.com/106390400/222943547-8e29728f-1b4f-4e86-a66f-c24db918e469.png">

 when I am in the lobby it does the obvious, It goes up by one

<img width="109" alt="image" src="https://user-images.githubusercontent.com/106390400/222943485-32b9a4eb-fcce-4712-902d-dfe343ae8add.png">

but when I go into my pixelmon server, it says there are no players playing.

<img width="95" alt="image" src="https://user-images.githubusercontent.com/106390400/222943530-148ce4e1-25bc-4b38-93f8-c33a0c49b5a1.png">

Whats weird is when I set the priority like this, It does the opposite. None playing in the lobby but when I go to the pixelmon server, it says 1 person playing. This is exactly why I made this issue here instead of Ambassador issues since I think it has nothing to do with it.

<img width="550" alt="image" src="https://user-images.githubusercontent.com/106390400/222943558-c72dfef7-548a-48b8-a369-a875369b607f.png">

My ping passthrough is set to none. I've tried it with and without the MiniMOTD plugin but I get the same result.

thanks

**Comment by Farmeurimmo** [comment]
Do your Lobby and Pixelmon servers send something abnormal in the logs or your velocity server ?

**Comment by Spreax** [comment]
This is the only Worrying thing that I guess you could say is concerning but I just think thats Ambassador doing its job.

<img width="841" alt="image" src="https://user-images.githubusercontent.com/106390400/223009067-04f9b10f-e90a-4dfb-91f8-c74689237987.png">

Other then this, No.

**Comment by Farmeurimmo** [comment]
Did you check this part of the configuration ? **If you can send what you have, I think that can help.**
![image](https://user-images.githubusercontent.com/71896330/223352800-91d4ab7d-f545-47da-87a5-a571ab7f5b36.png)

If you can use a small velocity plugin to display in console how much players are connected, you can check if the whole proxy don't detect them or if it's just a display bug.

**Comment by Spreax** [comment]
Noticed I had "Kick-existing-players" set to true so I set it to false but same result

But anyways this is my config 

<img width="597" alt="image" src="https://user-images.githubusercontent.com/106390400/223545399-d7135124-c574-406a-8e95-3fdcd7055124.png">

**Comment by Farmeurimmo** [comment]
Can you try with ping-passthrough="mods"?

**Comment by Spreax** [comment]
Unfortunately same result. I've tried all the options and still no luck.

**Comment by Farmeurimmo** [comment]
Did you try to make a plugin to check if the proxy is detecting players, for example when they change server with ServerConnectedEvent?

**Comment by Spreax** [comment]
If I'm honest I have no knowledge in making plugins and I cant find any plugins that do this. Velocity itself does give me this though

<img width="714" alt="image" src="https://user-images.githubusercontent.com/106390400/223762543-8bcd0fbb-14fd-4973-a6f9-03347c1d2446.png">

**Comment by Farmeurimmo** [comment]
Ok, I think I know now, you are connected to both servers for the proxy. You should try to remove the pixelmon server from the server to try to connect.

![image](https://user-images.githubusercontent.com/71896330/223789526-81815bd1-0906-4dd7-92b1-7abcd533f852.png)
Just use one server and tell me if that still append.

**Comment by Spreax** [comment]
Yeah it still happens. 

Don't know if its supposed to be like this but when i have 2 clients open and join the pixelmon server and then one on the lobby server, it allows me to do so. But when I try to have both clients in the same server, I get this.

<img width="688" alt="image" src="https://user-images.githubusercontent.com/106390400/223844367-e6ae13f6-b3e8-4923-83d4-942ade9aa312.png">

Its so strange to me because its almost like depending on the order in this image

![image](https://user-images.githubusercontent.com/106390400/223844726-0dca251c-cc69-4de1-99d6-31febdf2f895.png)

one server isnt actually connected to the proxy for some reason but lets me join.

**Comment by Farmeurimmo** [comment]
With try=["lobby"] it is still broken?

You shouldn't be able to connect to your server if he isn't registered in the config.

Did you check if your ip + port are correct for each server ?

**Comment by Spreax** [comment]
<img width="99" alt="image" src="https://user-images.githubusercontent.com/106390400/223861651-ca7c9472-4424-4da2-a7cb-8f018de4d3c4.png">

Yeah all the IP's and ports are correct

still nothing

I do have an issue where I cant connect to a vanilla server so I'm actually using 2 modded servers technically but vanilla players can connect to the lobby server which is using one mod to run ambassador. Ill bring up the issue in the ambassador github hopefully this will some how fix this issue as well ill let you know if it worked.

**Comment by Spreax** [comment]
Alright so ive been doing more testing. I've managed to fix the issue with the whole not being able to join vanilla server thing. I still do have this problem though.

I found out that the player count goes down when you swap servers. Not when you are in the lobby or pixelmon server depending on whatever the order is in this image.

![image](https://user-images.githubusercontent.com/106390400/224464486-6f8432b0-a389-43e8-8934-20da808ad1de.png)

 Lets say I am in the lobby and I switch to my pixelmon server. The player count does go down obviously as thats my issue. What is very strange is that when I join the lobby again but not through the multiplayer menu but by using "/server lobby" is that the player count does not go to one and stays at zero. This is very, very confusing.

Edit: Its ambassador 100%. I used 2 Vanilla paper servers and they both have the player count is at 1 no matter which one I join. Not sure what to do anymore since I absolutely need it to join the pixelmon server. I will take this up with the ambassador github but if you have any suggestions on what to do that would really help me.
